File: sysutils/cfengine3/patches/patch-src_unix.c
$NetBSD: patch-src_unix.c,v 1.1 2013/02/17 16:15:42 fhajny Exp $

Ignore HAVE_STRUCT_SOCKADDR_SA_LEN on NetBSD.

--- src/unix.c.orig	2013-01-11 12:54:39.000000000 +0000
+++ src/unix.c	2013-02-16 11:10:41.263683099 +0000
@@ -49,7 +49,7 @@
 
 #ifndef MINGW
 
-# ifdef HAVE_STRUCT_SOCKADDR_SA_LEN
+# if defined(HAVE_STRUCT_SOCKADDR_SA_LEN) && !defined(__NetBSD__)
 #  ifdef _SIZEOF_ADDR_IFREQ
 #   define SIZEOF_IFREQ(x) _SIZEOF_ADDR_IFREQ(x)
 #  else

File: sysutils/dbus-sharp/patches/patch-src_Unix.cs
$NetBSD: patch-src_Unix.cs,v 1.1 2013/06/11 02:29:40 obache Exp $

* Reverse the way the dllmap for libsocket is handled, by using libc
  https://github.com/mono/dbus-sharp/commit/1561585d601058392d5864e3add2bac3c67e14a7

--- src/Unix.cs.orig	2010-09-14 09:17:50.000000000 +0000
+++ src/Unix.cs
@@ -197,7 +197,7 @@ namespace DBus.Unix
 
 		// Solaris provides socket functionality in libsocket rather than libc.
 		// We use a dllmap in the .config to deal with this.
-		internal const string LIBSOCKET = "libsocket";
+		internal const string LIBSOCKET = "libc";
 
 		public const short AF_UNIX = 1;
 		// FIXME: SOCK_STREAM is 2 on Solaris
